site/public/js/pdf/PDFInitToolbar.js

```diff
@@ -176,9 +175,7 @@ window.onbeforeunload = function() {
     function handleToolbarClick(e){
         setActiveToolbarItem(e.target.getAttribute('value'));
     }
-    if(!loaded){
```

Comment:
I do not understand the logic behind loaded here. It seem like the toolbar is only enabled when loaded = false. If loaded = true, it seems like you get the inability to switch to drawing mode. Is there logic I'm missing?

Reply:
It's because the implementation for loading multiple PDFs is janky, in that some elements are destroyed, some are not. The toolbar is one of those things and so the this variable is there to prevent adding multiple event listeners to the toolbar elements. For example, without this, each time you load a new PDF (with reloading the page), hitting the zoom button will rotate the PDF by an extra 90 degrees.

Honestly, we're probably at a point where it would be better to just rewrite this whole thing to be cleaner implementation than just trying to fix minor bugs with it.